examples/sticky.ads.amp.html

```diff
@@ -160,7 +159,7 @@
     </div>
   </header>
   <main role="main">
-  <amp-sticky-ad layout="nodisplay">
+  <amp-sticky-ad layout="container">
```

Comment:
No need to specify `layout=container` - it's default. However, there's an issue here and why I'd like to go back to `nodisplay`. To start with, it does as as `nodisplay`. The second reason is that by the time the `amp-sticky-ad` code is done downloading the element might be actually visible, which could trigger all kind of issues that we don't want. Instead, we need to start with `nodisplay`, but as soon as `amp-sticky-ad` is initialized (`buildCallback`) to remove `display:none` and instead set `visibility:hidden`.